deployment/terraform/lambda-functions/alert_db/src/alert_db.py

```diff
@@ -0,0 +1,60 @@
+import json
+import logging
+import os
+
+import requests
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+ch = logging.StreamHandler()
+logger.addHandler(ch)
+
+ENVIRONMENT = os.getenv("ENVIRONMENT", "Staging")
+SLACK_BOT_WEBHOOK = os.getenv("SLACK_BOT_WEBHOOK")
+APP_NAME = "districtbuilder"
+SERVER_NAME = "DatabaseServer"
+
+DB_ALARM_NAMES = [
+    f"alarm{ENVIRONMENT}{SERVER_NAME}CPUUtilization-"
+    f"{APP_NAME}-{ENVIRONMENT.lower()}",
+    f"alarm{ENVIRONMENT}{SERVER_NAME}DiskQueueDepth-"
+    f"{APP_NAME}-{ENVIRONMENT.lower()}",
+    f"alarm{ENVIRONMENT}{SERVER_NAME}FreeableMemory-"
+    f"{APP_NAME}-{ENVIRONMENT.lower()}",
+    f"alarm{ENVIRONMENT}{SERVER_NAME}FreeStorageSpace-"
+    f"{APP_NAME}-{ENVIRONMENT.lower()}",
+]
+
+
+def handler(event, context):
+    if SLACK_BOT_WEBHOOK is not None:
+        for record in event["Records"]:
+            sns = record["Sns"]
+            message = json.loads(sns["Message"])
+            alarm_name = message["AlarmName"]
+            new_state = message['NewStateValue']
+            reason = message['NewStateReason']
+
+            if alarm_name in DB_ALARM_NAMES:
+                slack_message = {
+                    "text": f":alert::alert::alert: {alarm_name} state is "
+                    f" now {new_state}: {reason}\n"
+                    f"```\n{message}```"
+                }
```

Comment:
This looks great for when an alarm starts, but let's give a nicer/shorter message when the alarm goes away
```suggestion
                slack_message = {
                    "text": f":white_check_mark: {alarm_name} has recovered"
                    if new_state == "Ok"
                    else f":alert::alert::alert: {alarm_name} state is "
                    f" now {new_state}: {reason}\n"
                    f"```\n{message}```" if new_state
                }
```

Reply:
Agreed. I updated this based on your suggestions here: https://github.com/PublicMapping/districtbuilder/pull/1186/commits/a98aa05161e410214bb01fcb921bb17a1925afcf